config/initializers/pusher.rb

```diff
@@ -0,0 +1,7 @@
+require 'pusher'
```

Comment:
you are requiring pusher but not providing it.  You need to add `gem "pusher"` to your Gemfile.